aws/autoscaling.go

```diff
@@ -20,74 +18,80 @@ import (
 	"github.com/mozilla-services/reaper/state"
 )
 
-type AutoScalingGroupScalingSchedule struct {
-	enabled           bool
-	scaleDownString   string
-	scaleUpString     string
+type autoScalingGroupScalingSchedule struct {
+	Enabled           bool
+	ScaleDownString   string
+	ScaleUpString     string
 	previousScaleSize int64
 	previousMinSize   int64
 }
 
-func (s *AutoScalingGroupScalingSchedule) setSchedule(tag string) {
+func (s *autoScalingGroupScalingSchedule) setSchedule(tag string) {
 	// scalerTag format: cron format schedule (scale down),cron format schedule (scale up),previous scale time,previous desired size,previous min size
 	splitTag := strings.Split(tag, ",")
 	if len(splitTag) != 4 {
-		log.Error("Invalid Autoscaler Tag format %s", tag)
+		log.Error(fmt.Sprintf("Invalid Autoscaler Tag format %s", tag))
```

Comment:
go-logging has an `Errorf` function. It might be worth wrapping that in your reaperLog module instead of doing this.